src/docker.rs

```diff
@@ -17,7 +18,17 @@ const DOCKER: &str = "docker";
 const PODMAN: &str = "podman";
 
 fn get_container_engine() -> Result<std::path::PathBuf> {
-    which::which(DOCKER).or_else(|_| which::which(PODMAN)).map_err(|e| e.into())
+    let use_podman = env::var("CROSS_USE_PODMAN")
```

Comment:
Maybe the environment variable should rather be called `CROSS_CONTAINER_ENGINE` and contain the (path to the) command, i.e. `CROSS_CONTAINER_ENGINE=podman`?

Reply:
Good point. I'm changing it

Reply:
ready